ironic-deployment/ironic/ironic.yaml

```diff
@@ -26,6 +26,22 @@ spec:
                add: ["NET_ADMIN"]
           command:
             - /bin/rundnsmasq
+          livenessProbe:
+           exec:
+             command: ["sh", "-c", "curl --resolve 127.0.0.1:53 google.com"]
```

Comment:
dnsmasq here runs to serve as DHCP and TFTP server. It would be better to check those than the resolver

Reply:
Thanks, done.